### ISSUE TITLE
Paddle 2.0 下可直接使用 F.cross_entropy 简化代码

### ISSUE THREAD

**Issue by ShaneTian** [PaddlePaddle/PaddleSeg#861]
`F.cross_entropy` 相关 `mask` 代码：https://github.com/PaddlePaddle/Paddle/blob/ffa88c31c2da5090c6f70e8e9b523356d7cd5e7f/python/paddle/nn/functional/loss.py#L1249
`PaddleSeg CrossEntropyLoss` 有关 `mask` 的代码：https://github.com/PaddlePaddle/PaddleSeg/blob/8119f0dc907182a1ce2f2f46f84ad091c6f701e8/paddleseg/models/losses/cross_entropy_loss.py#L57

可直接使用 `F.cross_entropy` 中的 `ignore_index` 和 `reduction` 简化相关代码

**Comment by wuyefeilin** [comment]
感谢你的反馈，我们会充分考虑你的建议并进行相应的改进
